pkg/runtime/runtime.go

```diff
@@ -682,6 +726,7 @@ func (a *DaprRuntime) sendBindingEventToApp(bindingName string, data []byte, met
 		}
 
 		if err != nil {
+			go a.suspendInboundTrafficUntilAppAvailable()
```

Comment:
May I ask you why only for the GRPCProtocol this app availability check was added? What about HTTPProtocol ?

Reply:
@mtaghavi2005 See line 1258 and 1392.

Reply:
Thanks for the answer, but line 1258 and 1392 are related to Pub/Sub and what I meant was Input Binding.

Reply:
Maybe I missed that. I will add it to the pr. Thanks for noticing!